recipes/reaktoro/meta.yaml

```diff
@@ -0,0 +1,59 @@
+# Work in progress: using a commit hash from GitHub, as there's no tagged release yet
+{% set version = "0.1" %}
+
+package:
+  name: reaktoro
+  version: {{ version }}
+
+source:
+  url: https://github.com/reaktoro/reaktoro/archive/bb85d53b014fc8ab9aa22471be4f1dd4aa9b5a81.tar.gz
+  sha256: 65364d6eb1da26fd7aba61770c4deef4a7d6f1220799c362750a3f1ac89c8f89
+
+build:
+  number: 0
+  # NOTE: Linux is temporarily skipped because it'll need GCC 7
+  # (It will be added after a feedstock is created)
+  skip: True  # [py2k or not win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
```

Comment:
Please also add ``{{ compiler('c') }}``

Reply:
Hmm, this recipe doesn't need a C compiler, only C++, is it really necessary? Things seem to be working

Reply:
humm... my mistake, it is not necessary. I just saw you are using ``$CC`` and thought that you maybe was using the ``C`` compiler.I sent that messages using cellphone, haha
Yes, indeed that is not necessary, sorry